### DISCUSSION TITLE
TemplateLayoutPathResolver directory does not exist

### DISCUSSION THREAD

**Discussion by koitaki** [11ty/eleventy#1839] [Q&A] [answered]
Hi, anyone got any suggestions for how to fix/debug the following?

I get variations of this error:
**TemplateLayoutPathResolver directory does not exist for post.html: src/layouts
TemplateLayoutPathResolver directory does not exist for post: _includes**

There are a couple of issues related to it, but I wasn't able to apply them to my issue.
https://github.com/11ty/eleventy/issues/703
https://github.com/11ty/eleventy/issues/1211

The latter one suggests it was resolved by fixing layout paths, and I'd guess that's likely my issue.
Since it only started after I removed layouts from includes folder and made them separate.
I'm somewhat trying to follow Max Böck's Eleventastic approach and it obviously works fine for his site.
But to my mind I've tried the gamut of permutations, such as prefix underscores, using njk + front matter, etc.
And so far, zero luck.

In **eleventy.js** I have:
```
  dir: {
      input: 'src',
      output: 'dist',
      includes: 'includes',
      layouts: 'layouts',
      data: 'data'
  }
```

In the **src** folder, my folders are structured:
```
├── _data
├── _includes
├── _layouts
├── _includes
│   ├── contact-form.html
│   ├── meta-info.html
│   ├── page-head.html
│   ├── site-foot.html
│   ├── site-head.html
│   └── site-navigation.html
├── _layouts
│   ├── base.html
│   ├── collection.html
│   ├── home.html
│   ├── post.html
│   └── resource.html
├── admin
:
:
└── youtube-chaines
```

In my **post.html** layout template file:
```
{% extends base %}
{% block content %}
<article>
    <div class="[ wrapper ] [ flow ]">
        {{ content | safe }}
    </div>
</article>
{% endblock %}
```

In my **base.html** layout template file:
```
<!DOCTYPE html>
<html lang="en" class="no-javascript">
  <head>
    {% include "meta-info.html" %}
    <style>
    {% include "css/critical.css" %}
    </style>
  </head>
  <body>
    <div class="site-wrap">
      {% include "site-head.html" %}
      {% include "page-head.html" %}
      <main tabindex="-1" id="main-content" class="[ gap-top-200 md:gap-top-major ]">
        {% block content %}
        {% endblock %}
      </main>
      {% include "site-foot.html" %}
      {% include "site-scripts.html" %}
    </div>
  </body>
</html>
```

The debug error message seems to suggest its a general issue with the post.html template.
So I can't narrow down how to fix it.

Many thanks in advance for any advice/solutions!

**Comment by pdehaan** [comment, marked as answer]
Your directories seem to be prefixed with underscores on the file system, but in your config file, you dropped all the underscores so now eleventy presumably can’t find the directories or files

does this work?

```js
dir: {
      input: 'src',
      output: 'dist',
      includes: '_includes',
      layouts: '_layouts',
      data: '_data'
  }
  ```

**Reply by koitaki** [reply]
Yes. Yes it does work! 😁
Thanks very much again, Peter, appreciate your time and effort with the help.

During my hours of trying to debug, I had tried that, but other errors existed and I got lost in a fog of errors.
I've experimented with it since, and underscores are definitely included.
So now I've a better handle on template locations and re-reading the 11ty docs on config all makes sense 🙂

Cheers again!
